healdata.org/dashboard/Public/notebooks/cdc_wonder_demo.html

```diff
@@ -5460,7 +5460,7 @@
   content:"?"; }
 
 .bp3-icon-euro::before{
-  content:"�"; }
+  content:"�"; }

```

Comment:
Is this change intentional? I wonder if this change snuck in here somehow from a copy/paste or find/replace or an IDE auto-correcting something?

Reply:
oh, thanks for the catch, let me revert that